src/instruments/src/EFB/Components/Form/Input/Input.tsx

```diff
@@ -16,9 +16,11 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import React, {useEffect, useState} from "react";
+/* eslint-disable react/require-default-props */
```

Comment:
Let's just disable `react/require-default-props` in Typescript files (via overrides).

Reply:
Done